mettle/worker/worker.go

```diff
@@ -151,6 +148,7 @@ func runForever(instanceName, requestQueue, responseQueue, name, storage, dir, c
 	if err != nil {
 		return err
 	}
+	defer w.Shutdown()
```

Comment:
nit: Would it be worth expanding on the name here to make it clear what's being shut down? At first look my assumption was it shutdown the worker (which obviously didn't make sense).

Reply:
Yeah fair enough, I was thinking of it as being kind of generic but probably better just to be explicit.